pyproject.toml

```diff
@@ -2,3 +2,48 @@
 tortoise_orm = "db.db.TORTOISE_ORM"
 location = "db/migrations"
 src_folder = "./."
+
+[tool.pdm]
+distribution = false
+
+[project]
+name = "ib.py"
+version = "0.1.0"
+description = "Default template for PDM package"
+authors = [
+    {name = "ib-ai", email = ""},
+]
+dependencies = [
+    "aerich==0.7.1",
+    "aiohttp==3.8.3",
+    "aiosignal==1.3.1",
+    "aiosqlite==0.17.0",
+    "async-lru==1.0.3",
+    "async-timeout==4.0.2",
+    "asyncpg==0.27.0",
+    "attrs==22.1.0",
+    "charset-normalizer==2.1.1",
+    "click==8.1.3",
+    "dictdiffer==0.9.0",
+    "discord-py==2.1.0",
+    "emojis==0.7.0",
+    "frozenlist==1.3.3",
+    "idna==3.4",
+    "iso8601==1.1.0",
+    "multidict==6.0.3",
+    "pydantic==1.10.2",
+    "pypika-tortoise==0.1.6",
+    "python-dateutil==2.8.2",
+    "pytz==2022.6",
+    "six==1.16.0",
+    "SQLAlchemy==1.3.24",
+    "tomlkit==0.11.6",
+    "tortoise-orm==0.19.2",
+    "typing-extensions==4.4.0",
+    "yarl==1.8.2",
+    "ruff>=0.4.4",
```

Comment:
I think we should cut this down into just the dependencies we directly need (instead of also including dependencies of dependencies).

We don't have to pin the version numbers, since the lock file will deal with preserving dependency versions for us.

Reply:
This is a good point as well, I guess PDM just read in everything from `requirements.txt`.

It's also worth removing `requirements.txt` altogether since PDM will be handling all of this from now on?

Reply:
> It's also worth removing `requirements.txt` altogether since PDM will be handling all of this from now on?

Yes, `requirements.txt` should also be removed with this change.

Reply:
Ah yes most of this stuff just seems to be dependencies of discord-py